src/templates/uploads.html

```diff
@@ -182,17 +182,17 @@ <h3 title='{{ 'Toggle visibility'|trans }}' data-action='toggle-next' data-icon=
 
                   {% if mode == 'edit' %}
                     <!-- replace upload form -->
-                    <div id='replaceUploadForm_{{ upload.id }}' hidden>
-                      <form enctype='multipart/form-data' action='app/controllers/RequestHandler.php' method='POST'>
+                    <div id='replaceUploadForm_{{ upload.id }}' class='mt-1' hidden>
+                      <form enctype='multipart/form-data' action='app/controllers/RequestHandler.php' method='POST' class='form-inline replace-uploaded-file' data-long-name='{{ upload.long_name }}'>
```

Comment:
I suggest using a `data-action` instead of a css class for `replace-uploaded-file`. Also, now we end up with a form that is either sent "traditionally", or via ajax. Wouldn't the code be simpler if we opt for going full JS?

Reply:
> data-action instead of a css class for replace-uploaded-file

Agreed and done

> Wouldn't the code be simpler if we opt for going full JS?

I think it would be more complicated.